integration/tests/basic/helpers/assert_message.py

```diff
@@ -4,6 +4,7 @@
 class ErrorMessage(Enum):
     NEGATIVE_VALUE = "Resulting wei value must be between 1 and "
     INSUFFICIENT_FUNDS = "insufficient funds for transfer"
```

Comment:
we still need this  INSUFFICIENT_FUNDS?  Looks like you changed it for INSUFFICIENT_BALANCE

Reply:
we get INSUFFICIENT_FUNDS_ERROR when there are not enough neons on account, for example
and INSUFFICIENT_BALANCE_ERROR  when value in transaction is bigger then account balance